### ISSUE TITLE
Getting a 'BFLogInterceptor.h' file not found error on Xcode 14.3 and RN 0.71.6 wth RnBugfender v. 2.1.2 and BugfenderSDK v. 1.12.0

### ISSUE THREAD

**Issue by vlad-ps** [bugfender/rn-bugfender#70]
I'm getting a 'BFLogInterceptor.h' file not found error on Xcode 14.3 and RN 0.71.6 wth RnBugfender v. 2.1.2 and BugfenderSDK v. 1.12.0 on iOS. 

<img width="1195" alt="Screenshot 2023-04-05 at 2 20 22 PM" src="https://user-images.githubusercontent.com/75272184/230170049-9587019c-bd7c-4db9-baf3-1712c5ecb9a8.png">

This just started happening to me recently on Xcode 14.3 and RN 0.71.6. I can't find a workaround for this so far. 

EDIT: I found a workaround.

My iOS project builds without errors when I change line 20 in` RnBugfender.podspec` file from 

```
  s.dependency 'BugfenderSDK', '~> 1.10' #
```

to 

```
  s.dependency 'BugfenderSDK', '1.10' 
```

essentially pinning BugfenderSDK to v. 1.10.0 instead of downloading the latest v. 1.12.0. 

There is something not working with BugfenderSDK  v. 1.12.0 and RnBugfender v. 2.1.2. But everything compiles fine when I pin the SDK version to 1.10.0.

**Comment by franmontiel** [comment]
HI @vlad-ps, thanks for the report and the workaround

Some headers were not present in the latest release (1.12.0) by mistake. 
This has now been fixed and there is a new release (1.12.1) that works without issues.

Sorry for the inconveniences

**Comment by vlad-ps** [comment]
No worries! Thanks for the quick fix.
